src/custom/components/Tokens/TokensTableRow.tsx

```diff
@@ -51,16 +51,19 @@ const DataRow = ({
   toggleWalletModal,
 }: DataRowParams) => {
   const { account, chainId = ChainId.MAINNET } = useWeb3React()
+  const areThereTokensWithSameSymbol = useAreThereTokensWithSameSymbol()
 
   const theme = useTheme()
   const tradeLink = useCallback(
-    (token: Token, kind: OrderKind) => {
-      const inputCurrencyId = kind === OrderKind.SELL ? token.symbol : undefined
-      const outputCurrencyId = kind === OrderKind.BUY ? token.symbol : undefined
```

Comment:
All the links are pointing to sell orders only, so, removed the `kind` parameter